plugins/katello/3.14/installation/index.md

```diff
@@ -170,6 +170,74 @@ arguments. Multiple entries can be entered via
+
+When the foreman-installer is run, it is deployed with a predefined tuning profile named `default`. The definition of the `default` profile can be found in `/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml`.
+
+Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
+
+  * medium
+  * large
+  * extra-large
+  * extra-extra-large
+
+Definitions of various tuning profiles can be found in this directory `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`.  Based on your environment needs, use one of the tuning profiles in the installer as shown below.
+
+```bash
+foreman-installer --tuning medium
+```
+
+```bash
+foreman-installer --tuning large
+```
+
+```bash
+foreman-installer --tuning extra-large
+```
+
+```bash
+foreman-installer --tuning extra-extra-large
+```
+
+To reset to the default profile:
+
+```bash
+foreman-installer --tuning default
+```
+
+**Note**
+
+- Using the `--tuning` option does not update `/etc/foreman-installer/custom-hiera.yml`, instead it directly updates the required configuration as specified in the corresponding tuning profile. You can still use `custom-hiera.yml` to override any configuration if really needed.
+- If you had already used `custom-hiera.yml` and starting to use the tuned profiles, you may want to review the definition of tuned profiles (`/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`) and remove the duplicated configuration entries from your `custom-hiera.yml`.
+- Use `foreman-installer --help | grep tuning` to identify the current tuning level.
+
+Sample output for `default` tuning:
+
+```bash
+foreman-installer --help | grep tuning
+    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "default")
+```
+
+Sample output for `medium` tuning:
+
+```bash
+foreman-installer --help | grep tuning
+    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "medium")
+```
+
+It is difficult to find the exact tuning profile for a specific environment in the first attempt because it depends on various factors like the number of managed hosts, the features used in scale (E.g., Remote Execution), the bulk action on hosts, the total amount of content, amount of host traffic to foreman, etc.  Our recommendation is that you start with the tuning profile guidance as shown in the below table based on the number of managed hosts and scale up your environment as needed.
+
+**Note** The information in the table below is just a guidance.  It is strongly recommended that you monitor the foreman environment regularly and tune up as required.
```

Comment:
https://github.com/theforeman/foreman-installer/commit/8f8f2beaf46ed1658a28b3877a571054a5c5cec6 makes these a hard lower limits and need `--disable-system-checks` to skip. Maybe worth mentioning.

Reply:
good point, will add